README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Make sure to squash your changes locally before pushing, as the commit history created when developing the PR isn't relevant from a historical perspective.